cms/server/admin/handlers/base.py

```diff
@@ -305,6 +305,9 @@ def render_params(self):
         params["contest"] = self.contest
         params["url"] = self.url
         params["xsrf_form_html"] = self.xsrf_form_html()
+        # FIXME these are cheating
+        params["config"] = config
+        params["handler"] = self
```

Comment:
Is it me or the commit title doesn't agree much with what the commit does?

Reply:
The first line of the commit message is correct (config and handler are values that, under Tornado, were either automatically injected by Tornado or imported by templates). But, yes, the rest of the message has nothing to do with it and appears to be duplicated from the following commit. Not sure how that happened.